http/src/jsMain/kotlin/com.mirego.trikot.http/web/WebHttpResponse.kt

```diff
@@ -0,0 +1,33 @@
+package com.mirego.trikot.http.web
+
+import com.mirego.trikot.http.HttpResponse
+import org.khronos.webgl.ArrayBuffer
+import org.khronos.webgl.Int8Array
+import org.w3c.xhr.XMLHttpRequest
+
+fun getResponseBody(xhr: XMLHttpRequest): ByteArray? {
```

Comment:
Just a small detail, this could also be written as:
```kotlin
fun getResponseBody(xhr: XMLHttpRequest): ByteArray? = xhr.response?.let {
    Int8Array(it.response as ArrayBuffer, 0, it.length).unsafeCast<ByteArray>()
}
```